llvm/lib/Target/X86/GISel/X86InstructionSelector.cpp

```diff
@@ -1503,14 +1521,15 @@ bool X86InstructionSelector::materializeFP(MachineInstr &I,
   const Register DstReg = I.getOperand(0).getReg();
   const LLT DstTy = MRI.getType(DstReg);
   const RegisterBank &RegBank = *RBI.getRegBank(DstReg, MRI, TRI);
-  Align Alignment = Align(DstTy.getSizeInBytes());
+  // Create the load from the constant pool.
+  const ConstantFP *CFP = I.getOperand(1).getFPImm();
+  const auto &DataLayout = MF.getDataLayout();
```

Comment:
Usual convention call this DL, I dislike the shadowing of the type name (and last I checked lldb still struggled with this)